src/components/HTMLEngineProvider/HTMLRenderers/CodeRenderer.tsx

```diff
@@ -1,25 +1,32 @@
 import React from 'react';
+import type {TextStyle} from 'react-native';
 import {splitBoxModelStyle} from 'react-native-render-html';
-import _ from 'underscore';
+import type {CustomRendererProps, TText} from 'react-native-render-html';
 import * as HTMLEngineUtils from '@components/HTMLEngineProvider/htmlEngineUtils';
 import InlineCodeBlock from '@components/InlineCodeBlock';
 import useStyleUtils from '@hooks/useStyleUtils';
-import htmlRendererPropTypes from './htmlRendererPropTypes';
 
-function CodeRenderer(props) {
+type CodeRendererProps = CustomRendererProps<TText> & {
+    /** Key of the element */
+    key?: string;
+};
+
+function CodeRenderer({TDefaultRenderer, key, style, ...defaultRendererProps}: CodeRendererProps) {
     const StyleUtils = useStyleUtils();
     // We split wrapper and inner styles
     // "boxModelStyle" corresponds to border, margin, padding and backgroundColor
-    const {boxModelStyle, otherStyle: textStyle} = splitBoxModelStyle(props.style);
+    const {boxModelStyle, otherStyle: textStyle} = splitBoxModelStyle(style ?? {});
 
-    // Get the correct fontFamily variant based in the fontStyle and fontWeight
+    /**
+     * Get the defalult fontFamily variant
+     * */
     const font = StyleUtils.getFontFamilyMonospace({
-        fontStyle: textStyle.fontStyle,
-        fontWeight: textStyle.fontWeight,
+        fontStyle: undefined,
+        fontWeight: undefined,
```

Comment:
This was changed because this values were never correct. the library doesn't return any fontStyle or fontWeight in any platform, what it returns is the font itself but incomplete.
I've tested this in all platforms and in several different cases, what I realised is that the function always returned the default font because the variables were undefined.